docs/s3-bucket-versioning.md

```diff
@@ -0,0 +1,78 @@
+# **Steps to Create and Configure a Versioned S3 Bucket**
+
+## **Set a Variable for the Bucket Name**
+```bash
+BUCKET_NAME=temp-bucket-for-versioning
```

Comment:
Is that the standard name we use for our bucket?
What is the diff for a normal bucket and this one?

Reply:
No there's no standard names for the buckets afaik. This is just an example for a name. I'll update the doc specifying the user to pick a name according to the needs.